app/views/content/life-as-a-teacher/_directory-nav.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render Content::ImageBlockComponent.new(
+        title: "Life as a teacher",
+        title_color: "yellow",
+        image_path: "static/images/content/homepage/directory1.jpg"
+      ) do %>
+        <ul>
+          <li><a href="/life-as-a-teacher/pay-and-benefits">Pay and benefits</a></li>
```

Comment:
Having a static directory with constant links is useful in terms of maintenance but does mean you are linking to pages you are already on!

For example, this link is in the directory that appears on `/life-as-a-teacher/pay-and-benefits` - clicking on it just takes you to the top of the page you are already on

However, this is also the way that the footer works, so might not be an issue!

Reply:
Yeah I get this, but I think people will use it to navigate the life as a teacher section so i don't think its an issue